src/conversion/conversion.js

```diff
@@ -102,6 +114,347 @@ export default class Conversion {
 		};
 	}
 
+	/**
+	 * Sets up converters between the model and the view which convert a model element to a view element (and vice versa).
+	 * For example, model `<paragraph>Foo</paragraph>` is `<p>Foo</p>` in the view.
+	 *
+	 * `definition.model` is a `String` with a model element name to converter from/to.
+	 *
+	 *		// Simple conversion from `paragraph` model element to `<p>` view element (and vice versa).
+	 *		conversion.elementToElement( { model: 'paragraph', view: 'p' } );
+	 *
+	 *		// Override other converters by specifying converter definition with higher priority.
+	 *		conversion.elementToElement( { model: 'paragraph', view: 'div', priority: 'high' } );
+	 *
+	 *		// View specified as an object instead of a string.
+	 *		conversion.elementToElement( {
+	 *			model: 'fancyParagraph',
+	 *			view: {
+	 *				name: 'p',
+	 *				class: 'fancy'
+	 *			}
+	 *		} );
+	 *
+	 *		// Use `upcastAlso` to define other view elements that should be also converted to `paragraph` element.
+	 *		conversion.elementToElement( {
+	 *			model: 'paragraph',
+	 *			view: 'p',
+	 *			upcastAlso: [
+	 *				'div',
+	 *				{
+	 *					// Any element with `display: block` style.
+	 *					name: /./,
+	 *					style: {
+	 *						display: 'block'
+	 *					}
+	 *				}
+	 *			]
+	 *		} );
+	 *
+	 *		// `upcastAlso` set as callback enables a conversion of a wide range of different view elements.
+	 *		conversion.elementToElement( {
+	 *			model: 'heading',
+	 *			view: 'h2',
+	 *			// Convert "headling-like" paragraphs to headings.
+	 *			upcastAlso: viewElement => {
+	 *				const fontSize = viewElement.getStyle( 'font-size' );
+	 *
+	 *				if ( !fontSize ) {
+	 *					return null;
+	 *				}
+	 *
+	 *				const match = fontSize.match( /(\d+)\s*px/ );
+	 *
+	 *				if ( !match ) {
+	 *					return null;
+	 *				}
+	 *
+	 *				const size = Number( match[ 1 ] );
+	 *
+	 *				if ( size > 26 ) {
+	 *					return { name: true, style: [ 'font-size' ] };
```

Comment:
There should be a comment that this is information what should be consumed with the link to the full documentation. `name: true` looks complcated with no explanation.

Reply:
The comment might be helpful. I haven't added it cause, AFAIR, it is commented in `MatcherPattern` description.